src/go/rpk/pkg/cli/cmd/container/status.go

```diff
@@ -0,0 +1,29 @@
+package container
+
+import (
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/cli/cmd/container/common"
+	"github.com/spf13/cobra"
+)
+
+func newStatusCommand() *cobra.Command {
+	command := &cobra.Command{
+		Use:   "status",
+		Short: "Get status",
+		RunE: func(_ *cobra.Command, _ []string) error {
```

Comment:
@twmb Is there a reason to use RunE here? 

Maybe it's because of the docker client, exiting instead of returning an error might leave the client open but I'm not sure if this is the case.

Reply:
no good reason, but I didn't really want to shake the boat too much by trying to swap the existing RunE's for Run at this moment